roles/blobber/defaults/main.yml

```diff
@@ -0,0 +1,47 @@
+blobber_user: blobber
+
+blobber_cleanup: false # when set to "true" it will remove the container(s)
+
+################################################################################
+##
+## blobber container configuration
+##
+################################################################################
+blobber_container_name: blobber
+blobber_container_image: parithoshj/blobber:v0.0.1
+blobber_container_env: {}
+blobber_log_level: info
+
+# Network configs
+blobber_p2p_port: 20000
+blobber_announced_ip: "{{ ansible_host }}"
+blobber_api_listen_port: 20001
+blobber_container_ports:
+  - "{{ blobber_p2p_port }}:{{ blobber_p2p_port }}"
+  - "{{ blobber_p2p_port }}:{{ blobber_p2p_port }}/udp"
+  - "127.0.0.1:{{ blobber_api_listen_port }}:{{ blobber_api_listen_port }}"
+blobber_container_networks: []
+
+# Storage configs
+# Set blobber_pubkeys to "{{ ansible_inventory_sources[0] | dirname }}/files/validator_keys/{{ inventory_hostname }}/pubkeys.json" for devnets
+blobber_validator_pubkeys: |
+  ["0xpubkey","0xpubkey2"]
+blobber_datadir: /data/blobber
+blobber_validator_key_file: /data/pubkeys.json
```

Comment:
Doesn't the validator key file path also need to be passed as an arg in the `blobber_container_command`?

https://github.com/marioevz/blobber/blob/main/cmd/blobber.go#L83 Says `--validator-key-file` can be used for that. One thing that confused me though, was that the argument description there mentions:

> "Path to validator key file: List of validator keys, one per line in hex format 

We currently are using a JSON array format, so which one is true?